### PR TITLE
Aachen, Germany

### PR DESCRIPTION
**Pull request by sergiyprotsiv**
Should be a simple WFS dump to GML...

### CODE REVIEW

File: sources/de/nw/city_of_aachen.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+	"coverage": {
+		"country": "de",
+		"state": "nw",
+		"city": "Aachen" }
```
Comment (migurski) [inline sources/de/nw/city_of_aachen.json]: Looks like a stray closing JSON curly brace here.

**Comment by migurski** [comment]
Re-run this please

**Comment by sergiyprotsiv** [comment]
Not sure where the error is: the machine is getting an uncompressed gml file (without extension), but tries different extensions, says it has decompressed it, and then fails to find the gml inside...

**Comment by migurski** [comment]
I think it's a mis-map between the mime-type and an assumed filename extension. Definitely a bug in machine code.

**Pull request closed without merging**